### ISSUE TITLE
Z-Push backend urls possibly wrong

### ISSUE THREAD

**Issue by ponychicken** [mail-in-a-box/mailinabox#504]
First, thanks for this fantastic project. It took me a day to wrap my head around the DNS stuff, especially the custom rules the denic has for .de domains, but now my mailserver is running.

I could not get z-push to work immediately. I got the following errors in /var/log/z-push/z-push.log

````
25/07/2015 22:14:28 [ 1292] [WARN] [mail@domain.de] BackendCalDAV->Logon(): User 'mail@domain.de' is not authenticated on CalDAV
25/07/2015 22:14:28 [ 1292] [INFO] [mail@domain.de] AuthenticationRequiredException: Access denied. Username or password incorrect - code: 0
````

It turned out that the endpoint was wrong. Instead of `/cloud/remote.php/caldav/calendars/%u/`it was set to `/caldav/calendars/%u/`.

I believe this bug was introduced in https://github.com/mail-in-a-box/mailinabox/commit/9d6dc78b15f694fbf7281f5b853ad34f85d95407 where you moved owncloud from / to /cloud, but the endpoint urls weren't prefixed as well.

**Comment by JoshData** [comment]
No those URLs are supposed to be proxied, see:

https://github.com/mail-in-a-box/mailinabox/blob/master/conf/nginx-primaryonly.conf#L50

**Comment by ponychicken** [comment]
Yes I saw these lines. Using cadaver to try to connect to these proxied endpoint resulted in a 400 Bad Request error, only the direct url works for me here. I didn't change anything about the nginx config manually.

**Comment by JoshData** [comment]
This may be fixed by #537, which will be in the next release.
